test/expected/mysql_connector_query_test.out

```diff
@@ -0,0 +1,12 @@
+SELECT * FROM mysql_test_table;
+ id | varchar_example | text_example | integer_example | numeric_example |    timestamp_example     | boolean_example 
+----+-----------------+--------------+-----------------+-----------------+--------------------------+-----------------
+  1 | example         | example      |             100 |              10 | Fri Jan 01 00:00:00 2016 |               1
+(1 row)
+
+SELECT * FROM query_mysql_test_table;
+ count(1) 
+----------
+        1
+(1 row)
```

Comment:
Yes, right now are sorted but I didn't find a satisfying name. Maybe I change to use the NN_* convention